SchemaValidator/ValidationCommon.py

```diff
@@ -0,0 +1,122 @@
+import yaml
+import ast
+import urllib
+import json
+
+from sys import exc_info
+from urllib import request
+
+def setValidationStatus(uploadFileName=None, isValid=None, validationFeedback=None, token=None, url=None):
+    print('Setting upload validation status,\t' + 'uploadFileName=' + str(uploadFileName) + '\tisValid=' + str(isValid) + '\tvalidationFeedback=(' + str(validationFeedback) + ')\turl=' + str(url))
+    if (uploadFileName is None or isValid is None or validationFeedback is None):
+        print('Missing data, cannot set validation status:'
+              + '\tuploadFileName:' + str(uploadFileName)
+              + '\tisValid:' + str(isValid)
+              + '\tvalidationFeedback:' + str(validationFeedback))
+        return False
+    if(url is None):
+        url = getUrl()
+    if(token is None):
+        token = getToken(url=url)
+
+    validationSuccess = updateValidationStatus(fileName=uploadFileName, isValid=isValid, validationFeedback=validationFeedback, url=url, token=token)
+
+    return validationSuccess
+
+def updateValidationStatus(fileName=None, isValid=None, validationFeedback=None, token=None, url=None):
+    if(url is None):
+        url=getUrl()
+    if(fileName is None):
+        print('No filename was provided, I cannot set validation status.')
+        return False
+    try:
+        fullUrl = str(url) + '/api/uploads/setvalidation'
+        # TODO: Haml Files?
+        body = {'valid': isValid, 'validationFeedback': validationFeedback, 'fileName':fileName, 'type':'HML'}
```

Comment:
new object format is like 
{
'valid': true,
'validationFeedback': 'asdf',
'validator': 'MIRING',
'upload': {
  'fileName': 'asdf.hml'
  }
}

Reply:
Okay, making this change.